src/main.py

```diff
@@ -1,5 +1,22 @@
 def main():
-    print("Hewwo Shrine and Even Sky peole")
+    import random
+    seggs = (random.randint(1, 100))
+    guess = int(input("Gib me ur guess 1-100"))
+
+    while guess != seggs:
+
+        if guess < seggs:
+
+            print("ur lower")
+        elif guess > seggs:
+
+            print("ur higher")
+        guess = int(input("Gib me ur guess 1-100: "))
+
+    print("damn right")
+
+    #fsafafesf
+
```

Comment:
For your first time writing python code it is already pretty ok, going to reformat the code and move it into a different file to make it better fit with future changes.